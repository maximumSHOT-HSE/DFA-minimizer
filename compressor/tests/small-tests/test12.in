7 56 8
1 5 0
1 1 1
1 2 2
1 1 3
1 1 4
1 1 5
1 1 6
1 4 7
2 3 0
2 2 1
2 1 2
2 3 3
2 4 4
2 1 5
2 3 6
2 5 7
3 2 0
3 2 1
3 4 2
3 4 3
3 1 4
3 2 5
3 3 6
3 4 7
4 1 0
4 1 1
4 4 2
4 1 3
4 5 4
4 3 5
4 3 6
4 2 7
5 1 0
5 1 1
5 1 2
5 2 3
5 3 4
5 2 5
5 5 6
5 3 7
6 5 0
6 1 1
6 2 2
6 1 3
6 1 4
6 1 5
6 1 6
6 4 7
7 5 0
7 1 1
7 2 2
7 1 3
7 1 4
7 1 5
7 1 6
7 4 7
5
1
1 
