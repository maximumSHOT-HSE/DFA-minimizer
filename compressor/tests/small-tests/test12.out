5 15 3
3 1 0
5 1 0
2 1 2
5 1 2
4 2 0
2 3 0
3 3 1
4 3 1
3 3 2
1 4 0
4 4 2
1 5 1
2 5 1
5 5 1
1 5 2
1
2
4 5 
